lib/diversity/component.rb

```diff
@@ -251,13 +244,10 @@ def populate(hsh)
       @configuration.angular = hsh.fetch('angular', nil)
       # If set to true, use component name
       @configuration.angular = @configuration.name if @configuration.angular == true
-      @configuration.partials = hsh.fetch('partials', {})
-      @configuration.themes = Rake::FileList.new(hsh.fetch('themes', []))
-      @configuration.fields = hsh.fetch('fields', {})
-      @configuration.title = hsh.fetch('title', nil)
-      @configuration.description = hsh.fetch('description', nil)
-      @configuration.thumbnail = hsh.fetch('thumbnail', nil)
-      @configuration.price = hsh.fetch('price', nil)
+
+      #@configuration.title = hsh.fetch('title', nil)
```

Comment:
I don't like this way of disabling things. Either we intend to use the fields (then they should stay as they are) or we don't intend to use them (then they should be removed entiredly). Just commenting stuff out leaves much doubt about the **intent** of the code that is commented out.